ui/pages/confirmations/confirm-token-transaction-base/confirm-token-transaction-base.js

```diff
@@ -59,7 +59,7 @@ export default function ConfirmTokenTransactionBase({
 
   const ethTransactionTotalMaxAmount = Number(
     hexWEIToDecETH(hexMaximumTransactionFee),
-  );
+  ).toFixed(8);
```

Comment:
If this is matching a standard value in that hook, is it worth exporting a constant such as `ETH_DEFAULT_DECIMALS` to keep them aligned going forward?

Reply:
I updated the PR.